extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -123,6 +123,7 @@ export class AmpAnim extends AMP.BaseElement {
           }
           throw error;
         });
+    this.loadPromise_.then(() => this.updateInViewport_());
```

Comment:
This can can be handled without another promise instance by defining `#firstLayoutCompleted` to call `#updateInViewport_`.